kubejs/server_scripts/domum_ornamentum/recipes.js

```diff
@@ -0,0 +1,267 @@
+const registerDomumOrnamentumRecipes = (event) => {
+
+        event.remove({mod: 'domum_ornamentum', type: 'minecraft:crafting_shaped'})
+        event.remove({mod: 'domum_ornamentum', type: 'minecraft:crafting_shapeless'})
+
+//#region Extra Blocks
+         event.shaped(Item.of('domum_ornamentum:brick_extra', 5),
+         [
+             'B B',
+             ' B ',
+             'B B'
+         ], 
+         {
+             B: 'minecraft:bricks'
+         }).id('domum_ornamentum:brick_extra');
+
+         event.shaped(Item.of('domum_ornamentum:cobblestone_extra', 5),
+         [
+             'B B',
+             ' B ',
+             'B B'
+         ], 
+         {
+             B: '#forge:cobblestone'
+         }).id('domum_ornamentum:cobblestone_extra');
+
+         event.shaped(Item.of('domum_ornamentum:blue_cobblestone_extra', 8),
+         [
+             'BBB',
+             'BAB',
+             'BBB'
+         ], 
+         {
+             B: 'domum_ornamentum:cobblestone_extra',
+             A: '#forge:dyes/blue',
+         }).id('domum_ornamentum:blue_cobblestone_extra');
+
+         event.shaped(Item.of('domum_ornamentum:green_cobblestone_extra', 8),
+         [
+             'BBB',
+             'BAB',
+             'BBB'
+         ], 
+         {
+             B: 'domum_ornamentum:cobblestone_extra',
+             A: '#forge:dyes/green',
+         }).id('domum_ornamentum:green_cobblestone_extra');
+
+         event.shaped(Item.of('domum_ornamentum:purple_cobblestone_extra', 8),
+         [
+             'BBB',
+             'BAB',
+             'BBB'
+         ], 
+         {
+             B: 'domum_ornamentum:cobblestone_extra',
+             A: '#forge:dyes/purple',
+         }).id('domum_ornamentum:purple_cobblestone_extra');
+
+         event.shaped(Item.of('domum_ornamentum:mossy_cobblestone_extra', 8),
+         [
+             'BBB',
+             'BAB',
+             'BBB'
+         ], 
+         {
+             B: 'domum_ornamentum:cobblestone_extra',
+             A: 'tfc:plant/moss',
```

Comment:
tfc has I think 2 or 3 kinds of mosses, would it be better to use a tag instead? look at what the recipe for mossy cobble uses